api/envoy/config/trace/v3/opentelemetry.proto

```diff
@@ -19,9 +20,15 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 message OpenTelemetryConfig {
   // The upstream gRPC cluster that will receive OTLP traces.
   // Note that the tracer drops traces if the server does not read data fast enough.
-  // This field can be left empty to disable reporting traces to the collector.
+  // This field can be left empty to disable reporting traces to the upstream cluster.
+  // Only one of ``grpc_service``, ``http_service`` may be used.
   core.v3.GrpcService grpc_service = 1;
 
+  // The upstream HTTP cluster that will receive OTLP traces.
+  // This field can be left empty to disable reporting traces to the upstream cluster.
```

Comment:
Same as above.

Reply:
Done in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)